app/javascript/components/admin/instances/card.js

```diff
@@ -0,0 +1,105 @@
+import Deploy from './actions/deploy'
+import Restart from './actions/restart'
+import Stop from './actions/stop'
+import Delete from './actions/delete'
+import axios from 'axios'
+
+export default {
+  name: 'InstanceCard',
+
+  props: {
+    instance: Object
+  },
+
+  methods: {
+    updateStatus (data) {
+      this.status = data.status
+      this.processing = data.processing
+    }
+  },
+
+  data () {
+    return {
+      status: this.instance.status,
+      processing: false
+    }
+  },
+
+  render () {
+    return (
+      <div class='col-lg-4 col-sm-6 mb-4'>
+        <div class={`card card-instance-details border-${this.status.level}`}>
+          <div class='card-header'>
+            <div class='d-flex'>
+              <h5 class='card-title flex-grow-1 m-0'>
+                <a href={this.instance.url} class={`text-${this.status.level}`}>
+                  {this.instance.name}
+                </a>
+              </h5>
+              <div class='dropdown'>
+                <a
+                  class='dropdown-toggle'
+                  type='button'
+                  id={`openode-instance-${this.instance.id}`}
+                  data-toggle='dropdown'
+                  aria-haspopup='true'
+                  aria-expanded='false'
+                >
+                  <i class='fa fa-cog' />
+                </a>
+                <div
+                  class='dropdown-menu dropdown-menu-right'
+                  aria-labelledby={`openode-instance-${this.instance.id}`}
+                >
+                  <Deploy instance={this.instance} onUpdateStatus={this.updateStatus} />
+                  <Restart instance={this.instance} onUpdateStatus={this.updateStatus} />
+                  <Stop instance={this.instance} onUpdateStatus={this.updateStatus} />
+                  <div class='dropdown-divider' />
+                  <a class='dropdown-item' href={`/admin/instances/${this.instance.id}/scheduler`}>Scheduler</a>
```

Comment:
remove scheduler and docs

Reply:
Do you want to put these links anywhere else?

Reply:
for the docs, it should be on every page somewhere in the layout, see https://github.com/openode-io/openode-www/issues/70

depending on the context it will use the proper @doc_path

Reply:
for the schedule it can be in the settings section, https://github.com/openode-io/openode-www/issues/39